utils/grpc/runtime.go

```diff
@@ -24,7 +24,7 @@ func (g *RuntimeClientBuilder) WithContext(ctx context.Context, f func(ctx conte
 }
 
 type RuntimeServer struct {
-	DB_URL      string
+	DB_URL      map[string]string
 	RD_URL      string
```

Comment:
Same here, cannot match rd_url to the meaningful map of DB_URL above

Reply:
RD_URL string to RD_URL map[string]string ?

Reply:
No need. I thought it's read only. Ok to leave redis alone.